src/components/Map/Controls/index.tsx

```diff
@@ -172,6 +194,18 @@ export const MapControls = () => {
                   );
                 }}
               />
+              <DisposableLinkButtonComponent
+                href="https://oytutanak.com"
+                dispose={votingLocationsFilter.isOpen}
+                buttonLabel={t("links.sendDocumentTitle")}
+                icon={<SummarizeIcon />}
+              />
+              <DisposableLinkButtonComponent
+                href="https://t.me/oytutanakbot"
+                dispose={votingLocationsFilter.isOpen}
+                buttonLabel={t("links.sendDocumentMobileTitle")}
+                icon={<TelegramIcon />}
+              />
```

Comment:
Please make it responsive

![image](https://github.com/acikkaynak/musahit-harita-frontend/assets/76786120/8119699f-7a97-4829-9798-2982a01efff1)

Reply:
Forgot about disabling minWidth in smaller screens, sorry about that